vault/transit.go

```diff
@@ -76,3 +84,7 @@ func DecryptWithTransit(ctx context.Context, vaultClient Client, mount, key stri
 
 	return base64.StdEncoding.DecodeString(d.Plaintext)
 }
+
+func WithKeyVersion(v int) TransitOption {
```

Comment:
If we're going to have just the singular TransitOption (instead of TransitEncryptOptions, etc. for each operation), can we add a godoc comment to this explaining it's for Encrypt, so that future developers won't accidentally think they can pass WithKeyVersion to the Decrypt call, etc.?